manifestpy/mapmain.py

```diff
@@ -0,0 +1,33 @@
+def mapMainManifest(readfile, wtype):
+
+    mainOut = '{"@context":"http://schema.org",'
+    mainOut += '"@type":"' + wtype + '",'
+    if 'label' in readfile.keys(): mainOut += '"name:":"' + readfile['label'] + '",'
+    if 'attribution' in readfile.keys(): mainOut += '"provider":"' + readfile['attribution'] + '",'
+    if 'license' in readfile.keys(): mainOut += '"license":"' + readfile['license'] + '",'
+    if 'thumbnail' in readfile.keys(): mainOut += '"thumbnailURL":"' + readfile['thumbnail']["@id"] + '",'
+    if 'description' in readfile.keys(): mainOut += '"description":"' + readfile['description'] + '",'
+    metl = len(readfile['metadata'])
+    i = 0
+    while i < metl:
+        label = readfile['metadata'][i]['label']
+        value = readfile['metadata'][i]['value']
+        if label == 'Keywords':
```

Comment:
Since the manifests are created from a main.csv, and the main.csv is currently created by hand, you may consider at least comparing upper case to ensure a match.

e.g.
label = readfile['metadata'][i]['label'].upper()
if label == 'Keywords'.upper():  (or) if label == 'KEYWORDS':

This protects from the cases where a person may name the metadata in all lower case, mixed case, or upper case.

Reply:
I second Steve's review.